src/app/pages/LandingPage.svelte

```diff
@@ -208,6 +209,7 @@
 
       <button class="button button--primary button--shadow quick-start__submit" type="submit"> Upload </button>
     </form>
+    <GoogleSheetsInput />
```

Comment:
This is part of the `<form>` above, and should be placed _above_ the `<button type="submit">`